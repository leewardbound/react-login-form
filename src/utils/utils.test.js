import * as utils from '.';

describe('identity', function () {
  it('should return the exact value it\'s passed', function () {
    const primitive = 'foo';
    const nonPrimitive = () => {};

    expect(utils.identity(primitive)).toBe(primitive);
    expect(utils.identity(nonPrimitive)).toBe(nonPrimitive);
  });
});

describe('mq', function () {
  it('should render rules wrapped in a media querry', function () {
    expect(utils.mq('768px', `
      display: flex;
    `)).toMatchSnapshot();
  });
});

describe('cols', function () {
  it('should return proper CSS string', function () {
    expect(utils.cols(3, 12)).toMatchSnapshot();
  });
});;
