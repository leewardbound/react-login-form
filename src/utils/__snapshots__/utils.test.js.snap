// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`cols should return proper CSS string 1`] = `"calc((100% - 0.1px) / 12 * 3);"`;

exports[`mq should render rules wrapped in a media querry 1`] = `
"@media (min-width: 768px) { 
      display: flex;
     }"
`;
